src/commands/object/create.js

```diff
@@ -38,6 +38,8 @@ class CreateCommand extends BaseCommand {
 
     if (!data) {
       data = await this._readStdin();
+    } else if (typeof data === 'string') {
+      data = JSON.parse(data);
```

Comment:
First of all, add `try/catch`, in case it's not a JSON.
Also, I doubt this will work.

```
if (!data) {
  data = await this._readStdin();
} 

if (typeof data === 'string') {
  try {
    data = JSON.parse(data);
  } catch (err) {}
}

```

Reply:
Also, is the request method expecting a JSON object? To be checked

Reply:
Looking at this line https://github.com/warehouseai/wrhs/blob/master/src/utils/request.js#L58-L76 it is submitting as a JSON, so one would assume it would work w/ a JSON object that is being passed along

Reply:
It's submitting the entire body. `data` is just a field of that body, and it's string.

Reply:
sorry my mistake of the line. Though per docs it is expecting a JSON object to be passed as either stdin or `data`.